client/blocks/login/index.jsx

```diff
@@ -134,7 +134,7 @@ class Login extends Component {
 		}
 
 		return (
-			<LoginForm onSuccess={ this.handleValidUsernamePassword } />
```

Comment:
We still need to call `handleValidUsernamePassword` to ensure that users with 2FA enabled are redirected to the 2FA pages, otherwise it isn't possible to log in with a 2FA account.